src/Entity/User/User.php

```diff
@@ -12,4 +14,47 @@
  */
 class User extends BaseUser
 {
+    /**
+     * @var Collection|UserRole[]
+     * @ORM\OneToMany(targetEntity="UserRole", mappedBy="user", cascade={"persist", "remove"}, orphanRemoval=true)
```

Comment:
it works but i get `The field App\Entity\User\User#roles is on the inverse side of a bi-directional relationship, but the specified mappedBy association on the target-entity App\Entity\User\UserRole#user does not contain the required 'inversedBy="roles"' attribute.`

Reply:
fixed with `@ORM\AssociationOverrides` on the target side :tada: